_tuning-your-cluster/availability-and-recovery/workload-management/workload-management.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: Workload management
+nav_order: 62
+has_children: true
+parent: Availability and recovery
+---
+
+# Workload management
+
+Workload management allows users to group and search network traffic, isolating system resources to prevent the overuse of network resources by specific requests. It offers the following benefits:
+ 
+- Tenant-level admission control and reactive query management. It can identify and cancel resource-intensive queries when configured thresholds are exceeded, ensuring fair resource allocation.
+
+- Tenant-level isolation within the cluster for search workloads, operating at a node level.
+
+System administrators can dynamically manage query groups using the Workload management APIs. These query groups can be used to make search requests with resource limits.
+
+### QueryGroup
+
+```json
+{
```

Comment:
Not sure what this example does. Does it create a query group? If so, should there be an API operation attached to this?

Reply:
I replaced this with an example from the Query Groups page.

Reply:
I believe the original version is to show the schema of a query group instead of creating one